src/spec.ts

```diff
@@ -110,17 +110,17 @@ export interface GenericUnitSpec<E extends Encoding<any>, M> extends BaseSpec, L
   selection?: {[name: string]: SelectionDef};
 }
 
-export type UnitSpec = GenericUnitSpec<Encoding<Field>, Mark | MarkDef>;
+export type UnitSpec = GenericUnitSpec<Encoding<string | RepeatRef>, Mark | MarkDef>;
```

Comment:
Why not leave it as `Field` and have a link to one place where we explain what Field is? I think we should completely remove `Field` if you replace it here.

Reply:
We never use type as a union type in the documentation elsewhere. 

 It's quite a jump for readers to comprehend IMHO. 

This is the old docs, which makes the `Field` not really match the text.  

![pasted_image_9_13_17__10_26_am](https://user-images.githubusercontent.com/111269/30391389-011e66ee-986e-11e7-8f20-f06644b1bc67.png)

>  I think we should completely remove Field if you replace it here.

We still do the same in many other places. (Use the full union type in the schema, but keep the shorthand for references in the code.)